### ISSUE TITLE
🐧 Void Pyramid ↔ v18.6.1

### ISSUE THREAD

**Issue by stuartdb** [itchio/itch-compatibility-watchlist#124]
:rotating_light: game [Void Pyramid](https://willyelektrix.itch.io/void-pyramid) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/3c91d96eb727eea56a62db59d74482f6).

:running: 
Debian Testing x64
I had a quick look and noticed the crash was caused by missing dependencies. It looks like the game requires the 32 bit libraries, which would not be installed on most 64 bit systems unless the user has multi-arch enabled and already had a reason to install them. 

The linked debug log doesn't include the next few crashes I experienced, but I just kept installing the reported missing 32bit dependencies until the game launched.

On debian testing/jessie the libraries and the corresponding packages that were required to get the game to launch were:

| Lib | Package |
| --- | --- |
| libSDL-1.2.so.0 | libsdl1.2debian |
| libSDL_mixer-1.2.so.0 | libsdl-mixer1.2 |
| libXpm.so.4 | libxpm4 |

 :video_game:
To summarize, to run the game on Debian x64, do the following:
- Enable [multiarch](https://wiki.debian.org/Multiarch/HOWTO)
- Install dependencies:

`sudo apt install libsdl1.2debian:i386 libsdl-mixer1.2:i386 libxpm4:i386`
